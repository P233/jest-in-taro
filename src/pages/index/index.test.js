import HomePage from "./index.jsx";

describe("<HomePage />", () => {
  it("shallow HomePage", () => {
    const wrapper = shallow(<HomePage />);
    expect(wrapper).toMatchSnapshot();
  });

  it("mount HomePage", () => {
    const wrapper = mount(<HomePage />);
    expect(wrapper.find(".index").text()).toBe("Hello world!");
  });

  it("render HomePage", () => {
    const wrapper = mount(<HomePage />);
    expect(wrapper.find(".index").text()).toBe("Hello world!");
  });
});
