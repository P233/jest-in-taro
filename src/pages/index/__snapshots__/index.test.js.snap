// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> shallow HomePage 1`] = `
<Component
  className="index"
>
  Hello world!
</Component>
`;
